circle.yml

```diff
@@ -3,7 +3,7 @@ machine:
   services:
     - docker
   environment:
-    DOCKER_TAG: chronograf-20170127
+    DOCKER_TAG: chronograf-20170208
```

Comment:
@rossmcdonald has a docker build system for us.  Basically, it is a docker image we use that has all of our requirements like node, go, yarn, etc. 

The container is dated when it was last updated.  Recently, I asked Ross how the build worked and added build/README.md with notes about how to update the build system.